src/styles/components/expedition-tile.styl

```diff
@@ -0,0 +1,52 @@
+.tile
```

Comment:
I think this is altering the landing page reasons (see classNames in reasons.jsx) to look off?